tests/001-test-regular-volumes.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+test_regular_volumes(){
+  local test_status=1
+  local testname=`basename "$0"`
+  local errmsg="all snapshot destinations must be removed before removing the original volume"
+
+  docker volume create -d lvm --opt size=0.2G --name foobar 2>&1 >/dev/null
```

Comment:
Just asking to confirm: are you doing this (`2>&1 >/dev/null`, in that order) deliberately? As opposed to `>/dev/null 2>&1` or `&> /dev/null`? I can't tell from the code flow. If it's intentional, just ack with "yes I know the difference".

Reply:
Aah, you are right. It should be the other way around. Fixed, and changed at other places too.

Reply:
There can be a good reason to do it the original way: it hides stdout (e.g. non-useful info messages) but preserves stderr on the (old) stdout, which makes it possible to grep for errors. This is an infrequent case, which is why I wasn't sure. If you want to hide all output, it's usually cleaner to `&>/dev/null` which (on all shells I know of) redirects both stdout and stderr.